api/v1/mdbmulti/mongodb_multi_types.go

```diff
@@ -65,7 +65,7 @@ func (m *MongoDBMultiCluster) GetCredentialsSecretNamespace() string {
 }
 
 func (m *MongoDBMultiCluster) GetProjectConfigMapName() string {
-	return m.Spec.OpsManagerConfig.ConfigMapRef.Name
+	return m.Spec.GetProject()
```

Comment:
not sure if this is release notes worthy. It was a bug that caused that MongoDBMultiCluster could handle only `spec.opsManager` field, but in MongoDB we handle both `spec.opsManager` and `spec.cloudManager` via `GetProject()`